.env.example

```
@@ -1,95 +1,169 @@
-CRAWLER_DELAY=0.1
-CRAWLER_TIMEOUT=15.0
-CRAWL_MIN_WORDS=100
+MAX_CONCURRENT_CRAWLS=10
+CRAWLER_DELAY=0.05
+CRAWLER_TIMEOUT=5.0
+CRAWL_MIN_WORDS=50
 CRAWL_REMOVE_OVERLAYS=true
 CRAWL_EXTRACT_MEDIA=false
 CRAWL_KNOWLEDGE_GRAPH=false
-CRAWL_USER_AGENT="Crawlerr/0.1.0 (+https://github.com/user/crawlerr)"
+CRAWL_USER_AGENT=crawler-mcp/0.1.0 (+https://github.com/jmagar/crawler-mcp)
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The CRAWL_USER_AGENT key should go before the MAX_CONCURRENT_CRAWLS key

(UnorderedKey)